src/menu.js

```diff
@@ -0,0 +1,150 @@
+import {
+  mainFood,
+  salads,
+  drinks,
+  deserts,
+} from './menuObject';
+
+function menuPage(body) {
+  // const body = document.getElementById('content')
```

Comment:
It is better to delete commented code, for previous code use "git history" instead